src/shared/wallet/encode-locally.test.ts

```diff
@@ -0,0 +1,19 @@
+import { encodeForMasking, decodeMasked } from './encode-locally';
+
+describe.only('encode-locally.ts', () => {
+  test('encodeForMasking', () => {
+    const value = 'hello';
+    expect(decodeMasked(encodeForMasking(value))).toBe(value);
+  });
+
+  test('maskedValue is not equal to input value', () => {
+    const value = 'something';
+    expect(encodeForMasking(value)).not.toBe(value);
```

Comment:
May be it is slightly more reliable to check that encoded value does not include initial value?

Reply:
why not!